src/NHibernate.Test/NHSpecificTest/NH2302/Fixture.cs

```diff
@@ -152,6 +153,8 @@ public void BlobWithLength()
 				[Test]
 				public void BlobWithoutLength()
 				{
+					if (Dialect is MySQLDialect)
```

Comment:
It seems this one test fails only for MySql. There is another test here failing for another db, but for all other db it works. So maybe it is not that far of being fixed for all.